pkgs/by-name/da/das/package.nix

```diff
@@ -5,26 +5,28 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "das";
-  version = "0.3.8";
-  format = "pyproject";
+  version = "1.0.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "snovvcrash";
     repo = "DivideAndScan";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-a9gnEBTvZshw42M/GrpCgjZh6FOzL45aZqGRyeHO0ec=";
+    rev = "v${version}";
+    hash = "sha256-3d2sYxht4AMxz9K7Hm9VYWJ0bTfVFeGZFC+wGwteXCQ=";
+    fetchSubmodules = true;
   };
 
+  nativeBuildInputs = [
+    python3.pkgs.poetry-core
+    python3.pkgs.pythonRelaxDepsHook
+  ];
```

Comment:
```suggestion
```

Can be removed, same content will be identical line 24-27 as `pythonRelaxDepsHook` is not needed.